src/components/theme/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -19,13 +19,30 @@ const messages = defineMessages({
     id: 'Breadcrumbs',
     defaultMessage: 'Breadcrumbs',
   },
+  controlpanel: {
+    id: 'Controlpanel',
+    defaultMessage: 'Controlpanel',
```

Comment:
In English this text should be "Site Setup", since that is the title shown on /controlpanel